concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1144,6 +1144,34 @@ jobs:
 
                   cf push
 
+        - task: sync-admin-users
+          config:
+            platform: linux
+            image: docker:///ruby#2.2
```

Comment:
This would add other gems like aws-sdk and mail that shouldn't be in uaac container.

As discussed, the current solution is ok for now and we may revisit it in another story.